modules/preview-react/side-panel/stories/stories.tsx

```diff
@@ -40,7 +39,7 @@ export const Default = () => {
             }}
           >
             <AccentIcon style={{marginRight: 16}} icon={rocketIcon} />
-            <h3 {...labelProps} style={{...type.h4, color: colors.licorice500}}>
+            <h3 {...labelProps} style={{...type.levels.body.large, color: colors.licorice500}}>
```

Comment:
I think we've got an issue with anything that is `h4` from the beta heirarchy and `h3` or `h4` from the legacy hierarchy because they were previously headings but are now `body` so the font weight will drop.